server/models/Profile.js

```diff
@@ -4,7 +4,12 @@ const profileSchema = new mongoose.Schema({
   userId: {
     type: mongoose.Schema.Types.ObjectId,
     required: true,
-    ref: 'User'
+    ref: 'User',
```

Comment:
Please also rebase/adjust the base of the PR to only reflect relevant changes in this PR